modules/tinymce/src/themes/silver/main/ts/ui/core/complex/BespokeSelect.ts

```diff
@@ -179,14 +179,15 @@ const createMenuItems = (backstage: UiFactoryBackstage, spec: SelectSpec): Bespo
   };
 };
 
-const createSelectButton = (editor: Editor, backstage: UiFactoryBackstage, spec: SelectSpec, tooltipWithPlaceholder: string, textUpdateEventName: string): SketchSpec => {
+const createSelectButton = (editor: Editor, backstage: UiFactoryBackstage, spec: SelectSpec, tooltipWithPlaceholder: string, textUpdateEventName: string, btnName?: string): SketchSpec => {
```

Comment:
Seeing as no bespoke buttons can be made via public API, and we are now always passing in the btnName, should the btnName not be optional?